testtools/testcase.py

```diff
@@ -21,6 +22,8 @@
 import sys
 import types
 
+import six
+
```

Comment:
six is a third party import, so it should be grouped with the extras and unittest2 imports - but i see we have a little odd ordering here already, - perhaps put it with the wraps import below?